backend/src/main/java/com/woowacourse/levellog/authentication/application/OAuthService.java

```diff
@@ -1,34 +1,35 @@
 package com.woowacourse.levellog.authentication.application;
 
 import com.woowacourse.levellog.authentication.domain.OAuthClient;
-import com.woowacourse.levellog.authentication.dto.GithubCodeRequest;
+import com.woowacourse.levellog.authentication.dto.GithubCodeDto;
 import com.woowacourse.levellog.authentication.dto.GithubProfileDto;
-import com.woowacourse.levellog.authentication.dto.LoginResponse;
+import com.woowacourse.levellog.authentication.dto.LoginDto;
 import com.woowacourse.levellog.authentication.support.JwtTokenProvider;
 import com.woowacourse.levellog.member.application.MemberService;
 import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
 @Service
-@Transactional
+@Transactional(readOnly = true)
 @RequiredArgsConstructor
 public class OAuthService {
 
     private final MemberService memberService;
     private final OAuthClient oAuthClient;
     private final JwtTokenProvider jwtTokenProvider;
```

Comment:
```suggestion
    private final OAuthClient oAuthClient;
    private final JwtTokenProvider jwtTokenProvider;
    private final MemberService memberService;
```

클래스에서 사용되는 순서대로 필드 순서를 리팩터링하면 가독성을 아주 살짝 높일 수 있다고 생각해요!